src/set_theory/game/nim.lean

```diff
@@ -45,14 +45,9 @@ def nim : ordinal → pgame
           nim (ordinal.typein O₁.out'.r O₂) in ⟨O₁.out'.α, O₁.out'.α, f, f⟩
 using_well_founded { dec_tac := tactic.assumption }
 
-namespace pgame
-
-local infix ` ⧏ `:50 := lf
-local infix ` ≈ ` := equiv
-local infix ` ∥ `:50 := fuzzy
-local infix ` ≡r `:50 := relabelling
+open_locale pgame
 
-namespace nim
+namespace pgame
```

Comment:
Should `nim` be in the `pgame` namespace too?

Reply:
Yes, let me take care of that.